service/api.go

```diff
@@ -637,7 +630,12 @@ func (as *APIServer) validateProjectConfig(w http.ResponseWriter, r *http.Reques
 		ReadFileFrom: model.ReadFromLocal,
 	}
 	validationErr := validator.ValidationError{}
-	if _, projectConfig, err = model.LoadProjectInto(ctx, input.ProjectYaml, opts, "", project); err != nil {
+	if _, _, err = model.LoadProjectInto(ctx, validationConfigs.ParserProjectYaml, opts, "", project); err != nil {
+		validationErr.Message = err.Error()
+		gimlet.WriteJSONError(w, validator.ValidationErrors{validationErr})
+		return
+	}
+	if _, projectConfig, err = model.LoadProjectInto(ctx, validationConfigs.ProjectConfigYaml, opts, "", project); err != nil {
```

Comment:
Can we explicitly just call CreateProjectConfig here instead of going all the way through loadProjectInto again? And can we add a comment about why we need to do this and why we can't just use the one  returned from LoadProjectInto?